File: frontend/src/models/UserDTO.ts
class UserDTO{
    id: number;
    email?: string;
    fullName?: string;
    description?: string;
    verified?: boolean;
    role?: string;
    password?: string;
    image?: any;
    fromTime?: string;
    toTime?: string;

    constructor(id: number, email?: string, fullName?: string, description?: string, verified?: boolean, role?: string, password?: string, image?: any,
        fromTime?: string, toTime?: string){
        this.id = id;
        this.email = email;
        this.fullName = fullName;
        this.description = description;
        this.verified = verified;
        this.role = role;
        this.password = password;
        this.image = image;
        this.fromTime = fromTime;
        this.toTime = toTime;
    }
}

export default UserDTO

File: backend/src/main/java/com/serhiienko/backend/model/dto/UserDTO.java
package com.serhiienko.backend.model.dto;

import jakarta.persistence.Lob;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UserDTO {
    private Long id;
    private String email;
    private String fullName;
    private String description;
    private Boolean verified;
    private String role;
    @Lob
    private byte[] image;

    private String fromTime;
    private String toTime;
}
